docs/bigquery-usage.rst

```diff
@@ -34,31 +34,74 @@ Authorization / Configuration
      >>> client = bigquery.Client()
 
 - Override the credentials inferred from the environment by passing explicit
-  ``credentials`` to one of the alternative classmethods, `from_service_account_json`:
+  ``credentials`` to one of the alternative ``classmethod`` factories,
+  `:meth:gcloud.bigquery.client.Client.from_service_account_json`:
 
   .. doctest::
 
      >>> from gcloud import bigquery
      >>> client = bigquery.Client.from_service_account_json('/path/to/creds.json')
 
-  or `from_service_account_p12`:
+  or `:meth:gcloud.bigquery.client.Client.from_service_account_p12`:
 
   .. doctest::
 
      >>> from gcloud import bigquery
      >>> client = bigquery.Client.from_service_account_p12('/path/to/creds.p12', 'jrandom@example.com')
 
-- Override the project inferred from the environment by passing an explicit
-  ``project`` to the constructor, or to either of the alternative classmethods:
+
+Projects
```

Comment:
I believe listing projects via the RM API and this API should come back with the same results (just the RM API sends more details besides the ID + "friendlyName")

If you have a project role (ie, READER, OWNER, etc), then it'll show up in the RM "list projects" call, so the descriptions match.

I think that this API call only exists because of the RM API not existing at the time of publishing (BQ was a very early API -- existing before "Google Cloud Platform" was even a thing), and customers were asking for the list of projects that they have access to because of the "cross-project shared datasets" ability. IMO it doesn't belong here, and is confusing. I also expect that it would get ripped out in a future version, though I'll confirm that with someone on the BQ team. 

I really think that we should drop this method as it adds confusion ("is a BQ project different from what the rest of gcloud-python calls a Project?" "What does it mean to list projects when I set my project already?") and is just all sorts of wrong from a separation of API concerns perspective.